cli/src/commands/branch.rs

```diff
@@ -350,6 +350,20 @@ fn cmd_branch_rename(
         return Err(user_error(format!("Branch already exists: {new_branch}")));
     }
 
+    if view
+        .remote_branches_matching(
+            &StringPattern::exact(old_branch),
+            &StringPattern::everything(),
+        )
+        .any(|(_, remote_ref)| remote_ref.is_tracking())
+    {
+        writeln!(
+            ui.warning(),
+            "warning: Branch: {}, has remote branches which will not be renamed",
```

Comment:
nit: excessive comma?
and the parameter can be inlined as `"Branch {old_branch} ..."`

Reply:
Fixed.